bin/compile

```diff
@@ -35,16 +38,38 @@ cp -n $BP_DIR/scripts/Procfile $BUILD_DIR/Procfile
 echo "-----> Copying static.json to application slug"
 cp -n $BP_DIR/scripts/static.json $BUILD_DIR/static.json
 
-echo "-----> Installing Bower package manager"
 cd $BUILD_DIR
-npm install bower
+if [ -d "./bower_components" ]; then
```

Comment:
Maybe leave it with the backwards compat but add a deprecation explaining the optimazation? Maybe something like "In a future release, this buildpack will not install your bower components. Consider adding `bower install` to the `postinstall` section of `package.json` for optimal bower caching and future compatibility"? It's a... mouthful.